charts/images.yaml

```diff
@@ -194,7 +194,7 @@ images:
 - name: fluent-bit
   sourceRepository: github.com/gardener/logging
   repository: eu.gcr.io/gardener-project/gardener/fluent-bit-to-loki
-  tag: "v0.25.0"
+  tag: "v0.26.0"
```

Comment:
Well, you are not upgrading fluent-bit, but the custom plugin to this version. Please fix the release note.